src/common/pool_map.c

```diff
@@ -2295,6 +2294,7 @@ pmap_fail_node_get(struct pmap_fail_stat *fstat)
 
 	D_ASSERT(fstat->pf_node_nr < fstat->pf_node_total);
 	fnode = &fstat->pf_nodes[fstat->pf_node_nr++];
+	pmap_fail_node_init(fnode);
```

Comment:
[Nit] It seems this initialization is unnecessary, though doesn't hurt to do it.

Reply:
Never mind. I think there should normally be only a few failed nodes.

Reply:
right, not must to init here, as we inited when create fstat.